packages/controller/src/iframe/profile.ts

```diff
@@ -48,6 +49,16 @@ export class ProfileIFrame extends IFrame<Profile> {
       );
     }
 
+    if (policies?.contracts) {
```

Comment:
Hmm curious is this intended to be part of this PR?

Reply:
This will be temporary until we have something better with torii, but ultimately we will need correspondance between entrypoint hash and plain text (if it cannot be done through a RPC call to get the abi at the indexation time)